toonz/sources/common/tfx/ttzpimagefx.cpp

```diff
@@ -5,6 +5,10 @@
 
 #include "ttzpimagefx.h"
 
+#ifdef LINUX
+#define strtok_s strtok_r
+#endif
+
```

Comment:
strtok_s is used only for Windows, thus maybe just replace strtok with strtok_r for both MacOS X and Linux?